app/services/pul_metadata_services/client.rb

```diff
@@ -27,6 +27,13 @@ def retrieve_from_aspace_pulfa(id)
         response.body.dup.force_encoding("UTF-8")
       end
 
+      def retrieve_aspace_pulfa_ead(id)
+        conn = Faraday.new(url: "https://findingaids-beta.princeton.edu/catalog/")
```

Comment:
I'm leaving this findingaids-beta for now because fixing this across the code base will be a much bigger undertaking I just don't have time for right now.

Reply:
You're leaving the `retrieve_from_pulfa` method. I assume this is being used elsewhere and will be updated at a later date?

Reply:
Yes, it's being used in tests right now. It'll get removed later.